package.json

```diff
@@ -26,13 +26,14 @@
     "cls-hooked": "^4.0.1"
   },
   "devDependencies": {
-    "async": "1.5.2",
+    "async-1.5.2": "file:./test/stub-modules/async-1.5.2",
```

Comment:
Using local paths for dependencies is a regular npm feature described in npm docs, see https://docs.npmjs.com/files/package.json#local-paths

I would expect it to work on Windows too.